torch/ao/sparsity/experimental/data_sparsifier/lightning/callbacks/_data_sparstity_utils.py

```diff
@@ -0,0 +1,32 @@
+import logging
+from torch.ao.sparsity.experimental.data_sparsifier.base_data_sparsifier import SUPPORTED_TYPES
+
+logger: logging.Logger = logging.getLogger(__name__)
+
+
+def _create_data_sparsifier(data_sparsifier_args, data_sparsifier_type):
```

Comment:
this function feels a little bit redundant if it just takes 2 arguments and calls one with the other as an arg.

if not, should probably have arg order swapped to be more intuitive

Reply:
yeah, seems redundant - directly creating the object now